deploy/manifests/scc.yaml

```diff
@@ -0,0 +1,37 @@
+# Create SCC for Gateway resources
+kind: SecurityContextConstraints
+apiVersion: security.openshift.io/v1
+metadata:
+  name: nginx-ngf-admin
+allowPrivilegedContainer: false
+allowHostDirVolumePlugin: false
+allowHostIPC: false
+allowHostNetwork: false
+allowHostPorts: false
```

Comment:
I don't see `allowPrivilegeEscalation` on this list. Is that an option? I'm curious because we expose that value in our helm chart as a workaround, and it seems like we would want to include that option here as well.

Reply:
Ah this seemed to have slipped my eye, it is an option and I'll add it in once I test it doesn't break things.

Reply:
Unresolved this, if we decide to use a Helm template, this should most likely match the value as its changeable.

Reply:
We can leverage templating to only set this field if the helm value is true.

```
{{- if .Values.allowPrivilegeEscalation }}
allowPrivilegeEscalation: true
{{-  end }}
```